### PR TITLE
add social sharing buttons

### PR DESCRIPTION
**Pull request by lgomesp**
related to issue #20 

add social sharing buttons only on mobile, please check whether we are going to keep all those social medias or not.

### CODE REVIEW

**Comment by lgomesp** [comment]
can you please add the `hacktoberfest-accepted` label to the PR so it can be evaluated on Hacktoberfest? 😄

**Comment by Airthee** [comment]
Fine !

Can you please keep only the following socials : 

- email
- facebook
- linkedin
- reddit
- sms
- telegram
- twitter
- whatsapp

File: src/components/SocialSharingButtons.vue [hunk 1]
```diff
@@ -0,0 +1,238 @@
+<template>
+  <div class="social-container">
+    <h1>Share on</h1>
```
Comment (Airthee) [inline src/components/SocialSharingButtons.vue:3]: The text must be translatted (english and french).

In french : "Partager"  
In english : "Share on"

Locales are stored here : src/plugins/translation/

**Review by Airthee — CHANGES_REQUESTED** [review]

File: src/components/SocialSharingButtons.vue (hunk 1 above)
Comment (lgomesp) [reply src/components/SocialSharingButtons.vue]: > @lgomesp Fine !
> 
> Can you please keep only the following socials :
> 
> * email
> * facebook
> * linkedin
> * reddit
> * sms
> * telegram
> * twitter
> * whatsapp

This is done!

**Review by Airthee — APPROVED** [review]

**Review by Airthee — APPROVED** [review]

**Comment by Airthee** [comment]
@lgomesp Great !

One little more thing and we're done : can you move the SharingButtons component at the end of the footer ?  
Like that : 
image
image

File: src/components/SocialSharingButtons.vue [hunk 2]
```diff
@@ -0,0 +1,142 @@
+<template>
+  <div class="social-container">
+    <h1>{{ $i18n.t("home.share") }}</h1>
```
Comment (Airthee) [inline src/components/SocialSharingButtons.vue:3]: Finally I think this text is no longer necessary.  
Maybe you can delete this line and the associated translations 😉

File: src/plugins/translation/locale.en.json [hunk 3]
```diff
@@ -8,7 +8,8 @@
         "description": {
             "title": "How long, what the hell is this ?",
             "text": "You regularly forget your age or the age of a loved one ?<br>Need to know how long you have been with your partner ?<br>Add this website to your favourites and don't think anymore !<br>Just pick a date and see the difference grow instantly !<br>"
-        }
+        },
+        "share": "Share on"
```
Comment (Airthee) [inline src/plugins/translation/locale.en.json:12]: You can delete this.

File: src/plugins/translation/locale.fr.json [hunk 4]
```diff
@@ -8,7 +8,8 @@
         "description": {
             "title": "How long, qu'est ce que c'est ?",
             "text": "Il vous arrive d'oublier votre âge ou celui d'un(e) proche ?<br>Besoin de savoir depuis combien de temps dure une relation ?<br>Ajoutez ce site à vos favoris et arrêtez de réfléchir !<br>Renseignez votre date et voyez la différence instantanément !"
-        }
+        },
+        "share": "Partager"
```
Comment (Airthee) [inline src/plugins/translation/locale.fr.json:12]: You can delete this.

**Review by Airthee — CHANGES_REQUESTED** [review]
A tiny little more thing

**Review by Airthee — APPROVED** [review]

**Comment by Airthee** [comment]
Thank you for your contribution @lgomesp :+1:

**Comment by lgomesp** [comment]
Glad to help, thanks!

**Pull request merged**